slamon_afm/tests/models_tests.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime
+
+from slamon_afm.models import db, Task, Agent, AgentCapability
+from slamon_afm.tests.afm_test import AFMTest
+
+
+class TaskUnassignTests(AFMTest):
+    """
+    These tests are not actually testing any code but continuing
+    the tradition of using DB withing unittests by checking if the
+    SQLAlchemy declarations work as intended.
+    """
```

Comment:
But aren't the tests then testing the code, if the code being tested is the Model declarations ;)